packages/react-select/src/accessibility/index.ts

```diff
@@ -168,11 +160,6 @@ export const defaultAriaLiveMessages = {
       return `value ${label} focused, ${getArrayIndex(selectValue, focused)}.`;
     }
 
-    if (context === 'menu') {
-      const disabled = isDisabled ? ' disabled' : '';
-      const status = `${isSelected ? 'selected' : 'focused'}${disabled}`;
-      return `option ${label} ${status}, ${getArrayIndex(options, focused)}.`;
-    }
```

Comment:
Just clarifying, is this being removed because we're now using `aria-activedescendant` to convey the same information?

Reply:
Yes, you're right, now the user will be informed about disabled options because of aria-activedescendant: 'Blue unavailable 2 of 10'